locales/hr/json.json

```diff
@@ -1,34 +1,34 @@
 {
-    "(and :count more error)": "(and :count more error)",
-    "(and :count more errors)": "(and :count more errors)",
+    "(and :count more error)": "(i još :count greška)",
+    "(and :count more errors)": "(i još :count grešaka)",
     "30 Days": "30 dana",
     "60 Days": "60 dana",
     "90 Days": "90 dana",
     ":amount Total": "Ukupno :amount",
-    ":days day trial": ":Days day trial",
-    ":resource Details": ":Resource dijelovi",
-    ":resource Details: :title": ":Resource detalji: :title",
-    "A fresh verification link has been sent to your email address.": "A fresh verification link has been sent to your email address.",
-    "A new verification link has been sent to the email address you provided during registration.": "Nova veza za provjeru poslana je na adresu e-pošte koju ste naveli prilikom registracije.",
-    "A new verification link has been sent to the email address you provided in your profile settings.": "A new verification link has been sent to the email address you provided in your profile settings.",
-    "A new verification link has been sent to your email address.": "A new verification link has been sent to your email address.",
-    "Accept Invitation": "Prihvati Pozivnicu",
+    ":days day trial": "probno razdoblje od :days dana",
```

Comment:
A sentence must begin with a capital letter, unless the grammar rules of your language suggest otherwise.